### PR TITLE
Add openjdk upstream

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

**Comment by jerboaa** [comment]
Any idea why there are as many commits showing up? 42 of them for 4 files changed?

File: README.md [hunk 1]
```diff
@@ -90,6 +91,15 @@ API (and subsequently) website poll these repos to make binaries available.
 * [openjdk12-binaries](https://github.com/AdoptOpenJDK/openjdk12-binaries/) - AdoptOpenJDK nightly and release binaries for OpenJDK 12 (HotSpot VM and Eclipse OpenJ9 VM)
 * [openjdk-binaries](https://github.com/AdoptOpenJDK/openjdk-binaries/) - AdoptOpenJDK nightly and release binaries for OpenJDK (Latest) (HotSpot VM and Eclipse OpenJ9 VM)
 
+### OpenJDK binary repos
+
+These GitHub Repos are where builds from the OpenjDK update projects (lead by
```
Comment (jerboaa) [inline README.md]: Typo? `led` over `lead`

Comment (karianna) [reply README.md]: Fixed.

**Comment by karianna** [comment]
> Any idea why there are as many commits showing up? 42 of them for 4 files changed?

It's the way I pull in merge commits.  If you Squash and Merge the PR it will be fine.

**Comment by jerboaa** [comment]
Looks good to me.

**Comment by jerboaa** [comment]
> > Any idea why there are as many commits showing up? 42 of them for 4 files changed?
> 
> It's the way I pull in merge commits. If you Squash and Merge the PR it will be fine.

Alright, thanks.

**Pull request merged**